app/components/login.tsx

```diff
@@ -0,0 +1,117 @@
+'use client'
+import Link from 'next/link'
+import { useState, FormEvent } from 'react'
+import Image from 'next/image'
+import profile from './profile.svg'
+export default function Login_Form() {
```

Comment:
@mohd-khan09 Let's follow `LoginForm` naming like this for every component instead of using an underscore